lms/static/js/commerce/views/receipt_view.js

```diff
@@ -164,7 +164,7 @@ var edx = edx || {};
          * @return {object} JQuery Promise.
          */
         getCourseData: function (courseId) {
-            var courseDetailUrl = '/api/course_structure/v0/courses/%s/';
+            var courseDetailUrl = '/api/courses/v1/courses/%s/';
```

Comment:
I seem to remember that this API required a username querystring argument for signed-in users. Is that no longer the case?